### PR TITLE
corsix_th: bump version

### PR DESCRIPTION
**Pull request by hanya**

### CODE REVIEW

**Comment by hanya** [comment]
Related to #3594.

**Comment by Begasus** [comment]
Builds ok on gcc2h (secondary architecture), but the paths don't look quit ok to me (no share/bin should go into $appsDir)
corsix-th

File: games-simulation/corsix_th/corsix_th-0.62.recipe [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ADDITIONAL_FILES="corsix_th.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	corsix_th$secondaryArchSuffix = $portVersion
+	cmd:corsix_th$secondaryArchSuffix = $portVersion
```
Comment (Begasus) [inline games-simulation/corsix_th/corsix_th-0.62.recipe]: When using $prefix the binary will go into $prefix/bin and shouldn't get the $secondaryArchSuffux
```diff
-	cmd:corsix_th$secondaryArchSuffix = $portVersion
+	cmd:corsix_th = $portVersion
```

File: games-simulation/corsix_th/corsix_th-0.62.recipe [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+{
+	touch CorsixTH/LICENSE.txt
+	make install
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		$portDir/additional-files/corsix_th.rdef.in > $sourceDir/corsix_th.rdef
+	addResourcesToBinaries $sourceDir/corsix_th.rdef $binDir/corsix-th
```
Comment (Begasus) [inline games-simulation/corsix_th/corsix_th-0.62.recipe]: Change $binDir to $prefix/bin here and the next line (needed for secondaryArchitecture as mentioned above)

Comment (Begasus) [reply games-simulation/corsix_th/corsix_th-0.62.recipe]: Although these changes you made are valid for x86_64, for _x86 on gcc2h they are not

**Comment by Begasus** [comment]
LGTM, on my own experience it still didn't find the data files (not sure if there should be something changed in the buildsystem for it) but it looks much nicer now :+1:

**Comment by pulkomandy** [comment]
The data files are on the Theme Hospital CD, you need a copy to run this. It is expected that it asks where to locate them on first run.

**Comment by Begasus** [comment]
Thanks for the reply @pulkomandy thought they were (partial) provided in the package

**Comment by Begasus** [comment]
LGTM, maybe if someone has the datafiles to check this it could be merged?

**Comment by hanya** [comment]
I've tested with the data obtained from "the demo" in the following page: 
https://github.com/CorsixTH/CorsixTH/wiki/Getting-Started

**Comment by Begasus** [comment]
Thanks! Works ok (even for SDL2) :+1:

**Pull request merged**